webapp/channels/src/components/invitation_modal/invitation_modal.tsx

```diff
@@ -128,6 +131,11 @@ export default class InvitationModal extends React.PureComponent<Props, State> {
         this.setState({show: false});
     };
 
+    handleExit = () => {
+        focusElement(this.props.focusOriginElement, true);
```

Comment:
Shouldn't this be handled by the `GenericModal`? I thought it was handling restoring focus to the origin element